### PR TITLE
fix PercentTimeinGC metric

### PR DESCRIPTION
**Pull request by trunov-ms**
Fix bug for counting value PercentTimeinGC

From program `WEBMTest.exe` in class `Win32_PerfRawData_NETFramework_NETCLRMemory` I was found counter type
```
[DisplayName("% Time in GC"): ToInstance, countertype(537003008): ToInstance, 
perfindex(2606): ToInstance, helpindex(2607): ToInstance, defaultscale(0): ToInstance, 
perfdetail(100): ToInstance] uint32 PercentTimeinGC
```
The mapping for countertype(537003008) is *PERF_RAW_FRACTION*.

For Cooking Type MSDN has formula (100 * CounterValue) / BaseValue
[Article](https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/scripting-articles/ms974615(v=msdn.10)?redirectedfrom=MSDN#perf_raw_fraction
)

### CODE REVIEW

**Review by breed808 — CHANGES_REQUESTED** [review]
Hi there, thanks for submitting this and a big apology for not reviewing sooner! I must have accidentally hit "read" on the notification for this pull request and subsequently missed it.

I've left a couple minor comments, let me know if any info is needed.

File: collector/netframework_clrmemory.go [hunk 1]
```diff
@@ -144,6 +144,7 @@ type Win32_PerfRawData_NETFramework_NETCLRMemory struct {
 	NumberTotalcommittedBytes          uint64
 	NumberTotalreservedBytes           uint64
 	PercentTimeinGC                    uint32
+	PercentTimeinGC_base               uint32
```
[the remark this answers is not in the document]
Comment (trunov-ms) [reply collector/netframework_clrmemory.go:147]: According this [article](https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/scripting-articles/ms974615(v=msdn.10)?redirectedfrom=MSDN#perf_raw_fraction) we should use this calculation
` (100 * CounterValue) / BaseValue`

And "_base" value is described as 
>... BaseValue—well, BaseValue is just a number used to make the calculation come out right. To be honest, we don't know exactly how the BaseValue is derived. Fortunately, we don't need to know how it's derived as long as we can figure out what the value is—something we'll show you how to do in a second.

Do you want the comment in the code?

File: collector/netframework_clrmemory.go [hunk 2]
```diff
@@ -144,6 +144,7 @@ type Win32_PerfRawData_NETFramework_NETCLRMemory struct {
 	NumberTotalcommittedBytes          uint64
 	NumberTotalreservedBytes           uint64
 	PercentTimeinGC                    uint32
```
[the remark this answers is not in the document]
Comment (trunov-ms) [reply collector/netframework_clrmemory.go:146]: PercentTimeinGC is used in final calculation 
`float64(100*process.PercentTimeinGC)/float64(process.PercentTimeinGC_base)`

[the remark this answers is not in the document]
Comment (breed808) [reply collector/netframework_clrmemory.go:146]: Ah my mistake, I must have missed that when reviewing :facepalm:

File: collector/netframework_clrmemory.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (breed808) [reply collector/netframework_clrmemory.go:147]: I think a comment stating the base value is essentially a magic number would work. Linking to that article URL in the comment would also be good.

**Review by breed808 — APPROVED** [review]
Thanks for making those amendments. The file needs to be formatted with `gofmt` in order to pass CI.
Once that's done we can merge the changes.

**Pull request merged**